### DISCUSSION TITLE
Is this plugin being actively maintained? Offer to help

### DISCUSSION THREAD

**Discussion by fetwar** [mvdkwast/obsidian-copy-as-html#59] [General]
I really like this plugin! I have been using both this plugin and the more simple implementation under the same name (definitely not the same) from https://github.com/jenningsb2/copy-as-html 

However, the other plugin is no-longer being actively maintained and this plugin hasn't seen any response from the owner in ~7 months. I am hoping that it doesn't end up becoming unmaintained as well as the functionality these add is really important for Obsidian.

I would be happy to offer my help as a maintainer @mvdkwast

**Comment by mvdkwast** [comment]
Hi @fetwar, I'd be more than glad to have some help with this project. I will be away for 2 weeks, let's discuss how we do this when I come back. Thanks !

**Reply by mvdkwast** [reply]
Feel free to contact me at my handle @ gmail.com if you're still interested !

**Reply by fetwar** [reply]
Hi @mvdkwast, I've sent you an email :)
